content/posts/mobile_app_phase_one_report.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Mobile App: Phase One Report"
+date: 2020-07-09T08:51:49+05:30
+draft: false
+author: Nitish Aggarwal ([Nitish145](https://github.com/Nitish145))
+---
+
+The first evaluations happened from June 29th to July 3rd and Yay.. I Passed!!! This blog post comes as a summary to the amazing work done in the first phase of Google Summer of Code.
+
+<p align="center">
+	<img src="/img/first_eval_dashboard.png" alt = "First Evaluations Passed">
```

Comment:
Use markdown format for including images, html won't work https://deploy-preview-1--cvblog.netlify.app/posts/mobile_app_phase_one_report/

Reply:
On it.. Just checked no images were being rendered :P